civihr_employee_portal/features/civihr_employee_portal_features/civihr_employee_portal_features.info

```diff
@@ -48,13 +46,19 @@ features[field_instance][] = node-hr_documents-field_short_description
 features[menu_custom][] = hr-reports
 features[menu_custom][] = hr-reports-settings
 features[menu_links][] = hr-reports-settings_age-groups:reports/settings/age_group/
-features[menu_links][] = hr-reports_report-1-people:reports/people/
-features[menu_links][] = hr-reports_report-2-leave-and-absence:reports/leave_and_absence/
-features[menu_links][] = main-menu_civihr-admin:civicrm
-features[menu_links][] = main-menu_civihr-ssp:dashboard
+features[menu_links][] = hr-reports_leave-reports:reports/leave_and_absence/
+features[menu_links][] = hr-reports_people-reports:reports/people/
+features[menu_links][] = main-menu_help://civihr-documentation.readthedocs.io/en/latest/self-service-portal/login-screen/
```

Comment:
Any idea why this link was added now?

Reply:
Because it's present on the menu_links.inc file: https://github.com/compucorp/civihr-employee-portal/blob/staging/civihr_employee_portal/features/civihr_employee_portal_features/civihr_employee_portal_features.features.menu_links.inc#L95

In order for the features module to be able to manage it, it has to be present on both the menu_links.inc file and here in this .info file. If not, when we export the download the feature from the UI the menu item information will be removed from menu_links.inc.

Reply:
That makes sense, just was confused why it was added in this PR. I guess when all of the features were updated it caused a lot of changes that should have been in sync before.